utils/compose/compose.go

```diff
@@ -16,16 +16,18 @@ import (
 )
 
 type Compose struct {
-	Path     string              `yaml:"-"`
-	Version  string              `yaml:"version"`
-	Services map[string]*Service `yaml:"services,omitempty"`
+	Path     string                    `yaml:"-"`
+	Version  string                    `yaml:"version"`
+	Services map[string]*Service       `yaml:"services,omitempty"`
+	Networks map[string]*NetworkConfig `yaml:"networks,omitempty"`
```

Comment:
Why NetworkConfig not Network?

Reply:
Naming conflict between docker network definition and service network. Feel free to suggest something better.